pyfepdf.py

```diff
@@ -1812,7 +1812,7 @@ def main():
             # datos generales del encabezado:
             tipo_cbte = 19 if "--expo" in sys.argv else 201
             punto_vta = 4000
-            fecha = datetime.datetime.now().strftime("%Y%m%d")
+            fecha = '20210805' if "--fecha_prueba" in sys.argv else datetime.datetime.now().strftime("%Y%m%d")
```

Comment:
added this because the generated files always have the current date in them which can make testing with a saved file impossible. Adding this can ensure that the generated files have same date while testing.